src/store/Chat/index.js

```diff
@@ -1,35 +1,28 @@
 import API from '../endpoints';
 import makeDummy from '../dummy';
 
-// Flag to enable putting dummy data into redux
-// const DUMMY_DATA = process.env.NODE_ENV !== 'production';
-// const dummy = makeDummy(6, 50);
-
 const initialState = {
   activeId: '',
   activeIsOpen: null,
-  chats: {}, // DUMMY_DATA ? dummy.chatSessions : [],
-  messages: [], // DUMMY_DATA ? dummy.messages : [],
+  chats: [],
```

Comment:
When u fixed the conflicts I think u made this an `[]` instead of `{}` which I had changed it to